server/api/endpoints/shop_endpoints/orders.py

```diff
@@ -356,14 +351,14 @@ def patch(
 
 
 @router.put("/{order_id}", response_model=OrderUpdated, status_code=HTTPStatus.CREATED)
-def update(*, order_id: UUID, item_in: OrderUpdate, current_user: UserTable = Depends(auth_required)) -> OrderUpdated:
+def update(*, order_id: UUID, item_in: OrderUpdate) -> OrderUpdated:
     order = order_crud.get(order_id)
     if not order:
         raise HTTPException(status_code=404, detail="Order not found")
 
     if item_in.status and (item_in.status == "complete" or item_in.status == "cancelled") and not order.completed_at:
         order.completed_at = datetime.utcnow()
-        order.completed_by = current_user.id
+        # order.completed_by = current_user.id
```

Comment:
I think this was broken anyway, auth_required does not return a UserTable object, but a cognito token model